package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "start": "node .",
     "pretest": "jshint .",
-    "posttest": "nsp check"
+    "posttest": "nsp check",
+    "api": "node .",
+    "client": "./node_modules/.bin/webpack-dev-server --config server/webpack.dev.config.js"
```

Comment:
Do we even need this because of https://github.com/codefordenver/ketohero/blob/master/server/boot/01-frontend-server.js#L10 ?

I kind of like it as a separate process better, but I'm just curious.